apps/backend/src/plans/fiches/create-fiche/create-fiche.service.ts

```diff
@@ -52,12 +64,17 @@ export class CreateFicheService {
       );
     }
 
-    const ficheCree = await (tx ?? this.databaseService.db)
+    const [ficheCree] = await (tx ?? this.databaseService.db)
       .insert(ficheActionTable)
       .values(fiche)
       .returning();
 
-    return ficheCree[0]?.id;
+    const ficheId = ficheCree.id;
+    if (ficheId && ficheFields) {
+      await this.updateFicheService.updateFiche({ ficheId, ficheFields, user });
+    }
+
+    return ficheCree;
```

Comment:
pourrait être cool de commencer à suivre l'ADR et retourner des objets:

```suggestion
    return {success: true, data: ficheCree };
```